src/http/client.rs

```diff
@@ -27,17 +27,20 @@ impl HttpClient {
       .connect_timeout(Duration::from_secs(60))
       .user_agent("Tailcall/1.0");
```

Comment:
We should set them also via upstream settings. No hard coded values should be set over here, all should come from Upstream.

Reply:
@tusharmath Changes done. No hard coded values. But if the client doesn't pass any values then set default values.